splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/CheckBoxComponent.jsx

```diff
@@ -14,7 +24,7 @@ class CheckBoxComponent extends Component {
                 value={this.props.field}
                 onClick={this.handleChange}
                 disabled={this.props.disabled}
-                selected={!!this.props.value}
+                selected={!this.isFalse()}
```

Comment:
Can we move this `isFalse()` in util and pass value that we want to make boolean as argument?

Reply:
Added `isFalse`() and `isTrue()` utility functions in util